### PR TITLE
docs: explicitly set line wrap rules for error messages [skip ci]

### PR DESCRIPTION
**Pull request by yyforyongyu**

### CODE REVIEW

**Review by guggero — APPROVED** [review]

File: docs/code_contribution_guidelines.md [hunk 1]
```diff
@@ -465,9 +465,9 @@ to `gofmt` we've opted to enforce the following style guidelines.
      )
      ```
 
-Note that the above guidelines don't apply to log messages. For log messages,
-committers should attempt to minimize the of number lines utilized, while still
-adhering to the 80-character column limit.
+**Note that the above guidelines don't apply to log or error messages.** For
+log and error messages, committers should attempt to minimize the of number
```
Comment (Zero-1729) [inline docs/code_contribution_guidelines.md:469]: Minor nit (cont. below)

```suggestion
log and error messages, committers should attempt to minimize the number
```

File: docs/code_contribution_guidelines.md [hunk 2]
```diff
@@ -465,9 +465,9 @@ to `gofmt` we've opted to enforce the following style guidelines.
...
+lines utilized, while still adhering to the 80-character column limit.
```
Comment (Zero-1729) [inline docs/code_contribution_guidelines.md:470]:
```suggestion
of lines utilized, while still adhering to the 80-character column limit.
```

File: docs/code_contribution_guidelines.md (hunk 1 above)
Comment (yyforyongyu) [reply docs/code_contribution_guidelines.md]: Fixed, thanks for reviewing!

**Review by Zero-1729 — APPROVED** [review]
LGTM

**Pull request merged**